### ISSUE TITLE
@versioned is suppose to keep old record

### ISSUE THREAD

**Issue by mandeepsingh-private** [aws-amplify/amplify-cli#7655]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

12

### Amplify CLI Version

4.46.1

### What operating system are you using?

Popos

### Amplify Categories

api

### Amplify Commands

Not applicable

### Describe the bug

I was under the impression that using @versioned would keep the old versions, but I can't seem to find a way to access those old versions. So now I am assuming it simply controls version "clashing". If so, what is the recommended way of keeping older versions of objects?

### Expected behavior

When you version data, the old version of data is also kept in tabel. basic Change data capture. I am surprised how come we don't have it?

### Reproduction steps

Simply put @versioned the to schema in GraphQL and it does nothing.

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by cjihrig** [comment]
@mandeepsingh-private when you add `@versioned` to an `@model`, it should create a `version` field in the table. Are you saying that the `version` field does not exist?
